Topics/math-fundamentals.md

```diff
@@ -0,0 +1,172 @@
+# Math Fundamentals
+
+# What you will learn 
+In this guide, you'll learn what is the fundamental mathematical knowledge you need to have in order
+to build insightful Data Science projects and build machine learning systems upon them.
+
+You'll also learn that not so much is needed to start to apply yourself to real-world data problems!
+
+## Prerequisites
+The previous Paradiso guides.
+
+## Time to complete
+10 minutes.
+
+# Index
+- [Motivation](#Motivation)
+- [Courses](#Courses)
+  - [Calculus](#Calculus)
+  - [Linear Algebra and Matrix Algebra](#Linear-Algebra)
+  - [Optimization](#Optimization)
+- [Ask Questions](#Ask-Questions)
+- [Conclusions](#Conclusions)
+
+Let's dive right in!
+ 
+### Motivation
+
+Before anything else, watch [this](https://www.youtube.com/watch?v=VIbjHIGMjQM
+) and [this](https://www.youtube.com/watch?v=8CX-Q0gtSp8) awesome Ted Talks :-)
+
+**The entire field of Data Science is based on mathematics and statistics.**
```

Comment:
> ...mathematics, statistics and probability.
no?

Reply:
I assumed probability theory under statistics... what do you think?

Reply:
Statistics and probabilities are areas of maths, maybe one could write "The entire field of DS is based on mathematics and, more specifically, statistics and probabilities."
But I would actually omit the "probabilities" bit...

Reply:
agree with rodrigo